streaming/text/convert/README.md

```diff
@@ -0,0 +1,17 @@
+# Dataset preparation
+
+To use Streaming Dataset, We first convert the dataset from its native format to MosaicML's streaming dataset format (a collection of binary `.mds` files). Once in `.mds` format, we can store the dataset in a central location (filesystem, S3, GCS, etc.) and stream the data to any compute cluster, with any number of devices, and any number of CPU workers, and it all ~ just works ~ .
```

Comment:
should it be ~just works~?

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/77/commits/0a63fc68fb371e4b37fffe4ba043990b3816a9c4.